src/router/routes.ts

```diff
@@ -3,36 +3,59 @@
 // import StateExample from '@/views/StateExample.vue'
 import DefineCompany from '@/views/DefineCompany.vue'
 import ReviewConfirm from '@/views/ReviewConfirm.vue'
+import SigninView from '@/views/auth/Signin.vue'
+import SignoutView from '@/views/auth/Signout.vue'
 
 export const routes = [
   {
     path: '/',
     redirect: '/define-company'
   },
-  // {
-  //   path: '/home',
-  //   name: 'home',
-  //   component: Home,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
-  // {
-  //   path: '/MixinExample',
-  //   name: 'mixinExample',
-  //   component: MixinExample,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
-  // {
-  //   path: '/StateExample',
-  //   name: 'stateExample',
-  //   component: StateExample,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
+  {
+    path: '/signin/:idpHint',
+    name: 'signin',
+    component: SigninView,
+    props: true,
+    meta: {
+      requiresAuth: false
+    }
+  },
+  {
+    path: '/signin/:idpHint/:redirectUrl',
+    name: 'signin-redirect',
+    component: SigninView,
+    props: true,
+    meta: {
+      requiresAuth: false
+    }
+  },
+  {
+    path: '/signin/:idpHint/:redirectUrl/:redirectUrlLoginFail',
+    name: 'signin-redirect-fail',
+    component: SigninView,
+    props: true,
+    meta: {
+      requiresAuth: false
+    }
+  },
+  {
+    path: '/signout',
+    name: 'signout',
+    component: SignoutView,
+    props: true,
+    meta: {
+      requiresAuth: true
+    }
+  },
+  {
+    path: '/signout/:redirectUrl',
```

Comment:
1. How do we get to this route (page)?

2. Does this page require auth (see line 56)?

Reply:
This is used by the common-component in the header - there is a signout option there and it expects this route.  It requires authentication because it intends to remove the tokens from sessionStorage - however - is the common-component smart enough to handle a missing token gracefully ? Does it need to be a valid token ? These I do not know..